### DISCUSSION TITLE
HttpRequestNode parameter support file

### DISCUSSION THREAD

**Discussion by qqlww1987** [langgenius/dify#4788] [General]
I want node parameters to support files, what to do

**Comment by patryk20120** [comment]
Which node and what files are we speaking about?

**Reply by qqlww1987** [reply]
HTTP 请求

**Reply by patryk20120** [reply]
Well, for now there is no possibility of doing that if it comes to sending files over HTTP Request. But feel free to issue a feature request on that - I am sure that Dify team will take a look on that
